### ISSUE TITLE
possible MathJax optimizations

### ISSUE THREAD

**Issue by pkra** [readium/readium-shared-js#82]
Just to explain my comment at #76 

IIUC, the new MathJax integration waits 8 seconds for MathJax rendering to complete before unveiling the chapter (to reduce jitter). It might make sense to change the MathJax configuration to increase MathJax's chunking, i.e., the number of equations that get injected once they're finished rendering (see [EqnChunk etc](http://docs.mathjax.org/en/latest/options/SVG.html) ). 

Setting it very high will reduce the number of reflows triggered by MathJax inserting the output. That doesn't make sense is the user wants to start reading but if you're hiding it anyway, then this will offer a slight performance advantage.

I'm not sure how hard it is to play around with this without having automated the single-file build. Try loading the following before MathJax.js is loaded.

```
window.MathJax = {
    SVG: { EqnChunk: 50, EqnChunkFactor: 1.5, EqnChunkDelay: 100}
};
```

Those are the default values -- increase them to reduce reflows.

It's hard to say how many equations 8sec equate to since that depends on the complexity of the equations, the browser engine and most drastically on CPU speed. But a few hundred seem reasonable and might improve rendering speed.

For the record, we're also working on some speed improvements right now.
